scan/debian.go

```diff
@@ -305,7 +307,7 @@ func (o *debian) scanPackages() error {
 		return nil
 	}
 
-	if o.getServerInfo().Mode.IsDeep() || o.Distro.Family == config.Raspbian {
+	if o.getServerInfo().Mode.IsDeep() {
```

Comment:
I am worried about the behavior in raspbian && deep scan mode. 
I thought the following conditional is better.

```
if raspbian {
...
else If deep scan mdoe {
....
}
```

Reply:
In the case of raspbian && deep scan mode, I think that there is no problem because it does a deep scan and returns as usual. The reason why only raspbian is processed after the deep scan part is that the fast-root scan also needed to acquire the changelog.
Thanks to the review, I noticed that the scan mode part was made more obvious. You just need to explicitly show the fast-root scan for raspbian. So the code looks like this:
```
// fast-root scan for  raspbian
if !deep mode scan && raspbian {
}

if deep mode scan {
}
```

Reply:
LGTM